pkg/controller/clusteringress/clusteringress_controller.go

```diff
@@ -191,7 +192,7 @@ func (r *ReconcileClusterIngress) reconcileRoute(ctx context.Context, ci *networ
 	if err != nil && errors.IsNotFound(err) {
 		err = r.client.Create(ctx, desired)
 		if err != nil {
-			logger.Errorw("Failed to create OpenShift Route %q in namespace %q", desired.Name, desired.Namespace, err)
+			logger.Errorw(fmt.Sprint("Failed to create OpenShift Route %q in namespace %q", desired.Name, desired.Namespace), err)
```

Comment:
It's cleaner to just switch this to use `logger.Errorf` and drop the nested `fmt.Sprintf` instead of nesting `fmt.Sprintf` inside `logger.Errorw`. Also, was this tested? Because the `fmt.Sprint`on this line should really be `fmt.Sprintf` :)